aws/tf/modules/sra/credential.tf

```diff
@@ -77,21 +77,6 @@ resource "aws_iam_role_policy" "cross_account" {
             "aws:RequestTag/Vendor" : "Databricks"
         } }
       },
-      {
-        "Sid" : "AllowEc2RunInstanceImagePerTag",
-        "Effect" : "Allow",
```

Comment:
I don't believe so, because we're doing an explicit deny if not resource here https://github.com/JDBraun/terraform-databricks-sra/blob/d6fa97aa553b3209c911497f68f8dcf363dcd56a/aws/tf/modules/sra/credential.tf#L108

I haven't had any issues spinning warehouses or cluster without it